opensrp-anc/src/test/java/org/smartregister/anc/library/util/ANCFormUtilsTest.java

```diff
@@ -202,4 +221,38 @@ public void testFilterCheckboxValuesWhenFilterOptionSourceIsDefined() throws Exc
         assertTrue(stepThreePhysSymptomsPersist.getBoolean(ConstantsUtils.IS_FILTERED));
         assertTrue(stepThreePhysSymptomsPersist.has(ConstantsUtils.IS_FILTERED));
     }
+
+    @Test
+    public void testGetFormJsonCoreShouldReturnSameFormPassed() throws JSONException {
+        PartialContact partialContact = new PartialContact();
+        Mockito.when(partialContactRepository.getPartialContact(partialContact)).thenReturn(null);
+        Mockito.when(ancLibrary.getPartialContactRepository()).thenReturn(partialContactRepository);
+        ReflectionHelpers.setStaticField(AncLibrary.class, "instance", ancLibrary);
+        JSONObject form = new JSONObject(quickCheckForm);
+        JSONObject result = ANCFormUtils.getFormJsonCore(partialContact, form);
+        Assert.assertNotNull(result);
+        Assert.assertEquals(form.toString(), result.toString());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.assertEquals' due to existing static import 'org.junit.Assert.assertEquals'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=5117137)